docs/index.rst

```diff
@@ -13,7 +13,7 @@ Contents:
 
    building
    architecture
-   examples
+   docs/examples
```

Comment:
I think you meant to create an ``examples`` directory but instead created a second ``docs`` directory?